### ISSUE TITLE
The skip to section links icon has no discernible text

### ISSUE THREAD

**Issue by schalkneethling** [mdn/mdn-fiori#87]
The skip to section links next to heading on wiki pages receive focus, but have no accessible name, and have aria hidden set to true. <a href="#Basics" class="sectionLink"> 🔗</a>

1. Do not set aria-hidden to true
2. Add a label
3. Use a standard SVG icon from Dinocons

## Bugzilla

https://bugzilla.mozilla.org/show_bug.cgi?id=1563473
